tests/integration/targets/openssl_pkcs12/tasks/impl.yml

```diff
@@ -88,6 +88,14 @@
       path: '{{ output_dir }}/ansible_parse.pem'
       action: parse
       state: present
+    register: p12_dumped
+  - name: Dump PKCS#12 file again, idempotency
+    openssl_pkcs12:
+      src: '{{ output_dir }}/ansible.p12'
+      path: '{{ output_dir }}/ansible_parse.pem'
+      action: parse
+      state: present
+    register: p12_dumped_idempotency
```

Comment:
Can you replicate this task and add `check_mode: true` to it, and check whether it also is not changed?

Reply:
Done.